Services/CO.CDP.Organisation.WebApi/Model/Command.cs

```diff
@@ -93,7 +93,6 @@ public static class MappingExtensions
             StreetAddress = command.StreetAddress,
             StreetAddress2 = command.StreetAddress2 ?? "",
             Locality = command.Locality,
-            Region = "",
```

Comment:
This might need to eventually return. I saw Region used in the latest prototype.

Reply:
Yes introduced region in organisation API